monkey/infection_monkey/monkey.py

```diff
@@ -68,8 +79,7 @@ def __init__(self, args):
     def initialize(self):
```

Comment:
This class already has an initializer. It's called `__init__`. Why do we need another one?

Reply:
The `main.py` expects whatever class it's running to provide `initialize()`, `start()`, and `cleanup()`. The monkey and dropper classes both provide these. Changing this requires refactoring the dropper, which does basically nothing in its `initialize()` 
![image](https://user-images.githubusercontent.com/19957806/143863701-77f28c77-88ad-44d8-ab42-838a33a56f41.png)

I agree with @VakarisZ. Let's get rid of `initialize()` and use the constructor. `_teardown()` can just be `cleanup()` and can be called by `main.py` (like it is now.)